ow_regolith/include/RegolithSpawner.h

```diff
@@ -93,8 +93,8 @@ class RegolithSpawner
   double m_spawn_threshold;
   // Gazebo link name of the scoop particles will spawn in
   std::string m_scoop_link_name;
-  // a vector that describes the forward direction of the scoop
-  tf::Vector3 m_scoop_forward;
+  // vectors that describe the forward and down direction of the scoop
+  tf::Vector3 m_scoop_forward, m_scoop_down;
```

Comment:
The **m_scoop_down** seems to be a constant rather than an actual value. I would suggest removing and using the value directly in the function call 
```suggestion
  tf::Vector3 m_scoop_forward;
```

Reply:
related comment: https://github.com/nasa/ow_simulator/pull/214#discussion_r782345160

Reply:
Rather than making `m_scoop_down` a constant, I suggest I keep it as a member variable and expose it as a node parameter. That way users can use their own scoop models and will just have to reconfigure this node appropriately for the new model. The same should be done for `m_scoop_forward`, `m_scoop_link_name`, and `m_scoop_spawn_offset`. The parameters will be optional, and their default value will be the same as what's currently in the constructor.

Reply:
I don't know about that ..  to me this seems over engineering a workaround.

Reply:
https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it

Reply:
All of  m_scoop_forward, m_scoop_spawn_offset, m_scoop_link_name are constants.  My preference for encoding these, in order is:

1. as const file variables
2. as static class members

I do recommend abstraction over using the raw values in function calls.